content/concepts/xrpl-sidechains/cross-chain-bridges.md

```diff
@@ -0,0 +1,45 @@
+---
+html: cross-chain-bridges.html
+parent: xrpl-sidechains.html
+blurb: Cross-chain bridges for the XRP Ledger enable value in the form of XRP and other tokens (IOUs) to move efficiently between blockchains.
+status: not_enabled
+labels:
+  - Blockchain
+  - Interoperability
+---
+# Cross-Chain Bridges
+
+_(Requires the [XChainBridge amendment][] :not_enabled:)_
+
+Cross-chain bridges enable you to move XRP and tokens between the XRP Ledger and other blockchains. When referring to the blockchains connected by a bridge, one is the locking chain and the other the issuing chain.
+
+A locking chain is where the digital asset originates from. These assets are locked in a trust when sent across a bridge to an issuing chain.
+
+An issuing chain is an independent ledger with its own consensus algorithm and transaction types and rules. A wrapped version of the digital asset is minted and burned, depending on if an asset is received or sent from the locking chain.
```

Comment:
Should this mention a sidechain as an example?

Reply:
I don't think we need to mention it here, since we've got examples in the parent sidechains page.